src/docs/pages/component-docs/components/spacer/spacer-doc.component.ts

```diff
@@ -36,10 +29,7 @@ import { TabName } from '../../shared/scaffold/scaffold.component';
                 </div>
                 <div class="example-section">
                     <div class="example-heading">Spacer with Pixels</div>
-                    <div class="example-description">
-                        The <code>&lt;blui-spacer&gt;</code> also accepts a <code>height</code> and <code>width</code>
-                        input that specifies the number of pixels to consume.
-                    </div>
+                    <div class="example-description">A Spacer can also be given fixed dimensions instead of flex.</div>
```

Comment:
suggestion - The Spacer can also be given fixed dimensions instead of flex for static layouts.

Reply:
Reads well to me.